.github/workflows/dispatch-update-to-build-ci.yml

```diff
@@ -40,9 +32,9 @@ jobs:
           # get all the changed files in the push, only keep the parent directory (corresponds to the package name), and turn it into a space-separated list!
           echo "modified=$(git diff --name-only ${{ github.event.after }} ${{ github.event.before }} | cut -d '/' -f 2 | tr '\n' ' ')" >> $GITHUB_OUTPUT
```

Comment:
Do we still need this? It doesn't seem to be used.

Reply:
Also, do we want to run this for all branches? Is that what this is doing?

Reply:
No, it'll only run for the `main` branch. And yes, it won't be used it we remove the ability to created packages based on pushes to the branch name, although I can't think of a way to determine which models to be rebuilt based on tag.